chart/flux/README.md

```diff
@@ -211,191 +236,4 @@ Update Weave Flux version with:
 helm upgrade --reuse-values flux \
 --set image.tag=1.3.2 \
 weaveworks/flux
-```
-
-### Installing Weave Flux helm-operator and Helm with TLS enabled
```

Comment:
@stefanprodan Are you OK with moving this to a separate section, namely `site/helm-operator.md`?